lib/ansible/modules/network/nxos/nxos_interface.py

```diff
@@ -279,7 +279,7 @@ def get_interface(intf, module):
 
     command = 'show interface {0}'.format(intf)
     try:
-        body = execute_show_command(command, module)[0]
+        body = run_commands(module, [command])[0]
```

Comment:
This command expects results in JSON format (the `TABLE_interface` bits below), so needs to have output set to json